src/app/js/admin/api/job.js

```diff
@@ -0,0 +1,21 @@
+import fetch from '../../lib/fetch';
+import { getJobLogsRequest, getCancelJobRequest } from '../../user';
+
+export const getUserLocalStorageInfo = () =>
+    JSON.parse(window.localStorage.getItem('redux-localstorage').user);
+
+const getJobLogs = (queue, jobId) => {
+    const state = getUserLocalStorageInfo();
+
+    const request = getJobLogsRequest(state, queue, jobId);
```

Comment:
What's in the state? Why do we send the full state? Can't we cherry-pick only what we need from the state?